app/networkapi/urls.py

```diff
@@ -25,7 +25,6 @@
 urlpatterns = [
     url('^admin/', include(admin.site.urls)),
     url(r'^api/people/', include('networkapi.people.urls')),
-    url(r'^api/features/', include('networkapi.features.urls')),
```

Comment:
inconvenient as it sounds, let's not overload PRs: remove features in a different PR